iOSCoreSample/iOSCoreSample/AppDelegate.swift

```diff
@@ -0,0 +1,40 @@
+import PushKit
+import RealtimeKit
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate, PKPushRegistryDelegate {
+    var window: UIWindow?
+    var callManager: CallManager!
+    var meeting: RealtimeKitClient!
+
+    func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        callManager = CallManager()
```

Comment:
Let's keep this sample focused on RealtimeKitCore. Please remove the CallManager and PushKit integration.